zokrates_js/build.rs

```
@@ -0,0 +1,45 @@
+use std::path::Path;
+use std::{env, fs};
+use walkdir::WalkDir;
+
+fn main() {
+    export_stdlib_js();
+    export_metadata();
+}
+
+fn export_stdlib_js() {
+    let root = "../zokrates_stdlib/stdlib";
+    let mut stdlib = json::JsonValue::new_object();
+
+    for entry in WalkDir::new(root)
+        .into_iter()
+        .filter_map(Result::ok)
```

This ignores errors, instead it should probably throw if one happens: `.map(Result::unwrap)`